raiden/network/transport/matrix/client.py

```diff
@@ -226,6 +227,14 @@ def create_room(
     def get_presence(self, user_id: str) -> Dict[str, Any]:
         return self._send("GET", f"/presence/{quote(user_id)}/status")
 
+    def invite(self, room_id: RoomID, offer: Dict[str, str]) -> None:
+        self.send_message(room_id=room_id, text_content=json.dumps(offer), msgtype="m.call.invite")
```

Comment:
@andrevmatos specifically asked for it. Lets align with him first.

Reply:
From [Server behavior](https://matrix.org/docs/spec/client_server/latest#server-behaviour), the servers (per spec) aren't instructed to drop unknown `msgtype`, as long as it has a string `body` (may be empty). From my PoV, the actual value of these keys are _per protocol_, and it should be fine to send custom payloads while complying with the more general rules. We may discuss it further if needed, but we will have to adapt our implementation, and this would ease the work for us.

Reply:
This sounds like needlessly setting ourselves up for future pain.

If Synapse (or another homeserver implementation we may want to use in the future) ever start validating the `msgtype` attribute we'll have a quite difficult time changing this while remaining backwards compatible.